services/convert/release_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// Copyright 2023 The Forgejo Authors. All rights reserved.
```

Comment:
Hi @earl-warren please remove this line, we have given this feedback to you previously, and also is stated what new files should have in our contributing.md. This is following advice given to us from The Linux Foundation.
Many others are able to contribute to the project following the contributing.md, and we don't have // Copyright Allspice, //Copyright Blender, //Copyright ... etc.. there are literally hundreds of orgs that contribute to this project.

The project appreciates you upstreaming some of the code, however, if you aren't able to follow the contribution documentation we can't make a special exception just for you. By continuously knowingly ignoring the contributing documentation we may need to take action.

Reply:
Without wishing to comment on this particular instance, I should note that _the one requirement_ of [the MIT license](https://github.com/go-gitea/gitea/blob/main/LICENSE) is that the copyright notice is preserved. It is forbidden to remove it without permission from the author.

If the author chooses to contribute directly to Gitea under Gitea's copyright notice, that's obviously their choice. If they contribute under a different copyright notice, it is illegal to remove it as by doing so you are no longer covered by the license and are therefore violating that author's copyright by copying their code.